app/views/devise/sessions/new.html.haml

```diff
@@ -0,0 +1,17 @@
+%h2 Log in
+= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
+  .field
```

Comment:
please remove this file, no need to have it in the code base

Reply:
Don't we need it for our login feature?

Reply:
nope, try removing it and run the tests

Reply:
NO we CAN'T!!! If we do, our sign up function is removed along with it.